WordPress/Classes/Utility/Blogging Reminders/PromptRemindersScheduler.swift

```diff
@@ -70,64 +135,29 @@ class PromptRemindersScheduler {
                 && (!prompt.inSameDay(as: currentDate) || reminderTime.compare(with: currentDate) == .orderedDescending)
 
             }.forEach { promptToSchedule in
-                let _ = self.scheduleNotification(for: promptToSchedule, blog: blog, at: reminderTime)
+                let _ = self.addLocalNotification(for: promptToSchedule, blog: blog, at: reminderTime)
             }
 
             // TODO: Save notification identifiers to local store.
 
             // TODO: Schedule static notifications.
 
+            print("[BloggingPrompt] Prompt reminder scheduled!")
```

Comment:
I don't think we'll be wanting this. 😄

Reply:
Ah, good catch! 🦅 👀 . This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/18705/commits/881c62ff8143b6081170069aafa6b41eb3fff481.